css/main.css

```diff
@@ -1,3 +1,45 @@
-/* Add your custom CSS here */
-
-/* Dont' forget to link this file to your HTML in the <head> */
+.main-content {
+  /* background-color: transparent; */
+  position: absolute;
+  top: 8rem;
+  left: 7%;
+  width: 93%;
+}
+
+.main-background {
+  height: 1050px;
+}
+
+.main-background div:nth-child(1) {
+  background-color: white;
+  height: 50%;
+  width: 100%;
+}
+
+.main-background div:nth-child(2) {
+  background-color: #faefe9;
+  height: 50%;
+  width: 100%;
+}
+
+@media (min-width: 768px) {
+  .main-content {
+    margin-left: 0;
+    margin-right: 0;
+    top: 14rem;
+    left: 10%;
+    width: 80%;
+  }
+
+  .main-background div:nth-child(1) {
+    background-color: #faefe9;
+    width: 46%;
+    height: 100%;
+  }
+
+  .main-background div:nth-child(2) {
+    background-color: white;
+    width: 54%;
+    height: 100%;
+  }

```

Comment:
Good work making the dark beige background work on mobile and on desktop 👍 Watch out for the height of that background on mobile though - the height is not adapting to the content. It's a little tricky to get this to work, have a look at [my sample solution here](https://codesandbox.io/s/resume-layout-solution-j1tw5?file=/index.html).

Reply:
Also, using combined selectors like this `div:nth-child(2)` is fine here, but can get complicated when you have more complex markup. Don't hesitate to add your own custom class here for this (like I did with 'pinned-background')